lib/swagger_client/version.rb

```diff
@@ -3,13 +3,13 @@
 
 #No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
 
-OpenAPI spec version: 1.0.32
+OpenAPI spec version: 1.0.33
 
 Generated by: https://github.com/swagger-api/swagger-codegen.git
 Swagger Codegen version: 2.4.19
 
 =end
 
 module SwaggerClient
-  VERSION = '1.0.32'
+  VERSION = '1.0.0'
```

Comment:
This is the line that resets when you run the script. Line 14 should be `VERSION = '1.0.33'`